plugins/modules/nd_setup.py

```diff
@@ -248,6 +255,31 @@
                 - The ASN of the BGP peer.
                 type: int
                 required: true
+  deployment_mode:
+    description:
+    - This is used for enabling the available services between
```

Comment:
- This is used for enabling the services between Orchestrator, Fabric Controller and Insights during the initial installation.

Reply:
Better phrasing thanks, will fix it!